platforms/cosmwasm/src/platform.ts

```diff
@@ -128,6 +130,7 @@ export class CosmwasmPlatform<N extends Network>
   }
 
   static async getBalances<C extends CosmwasmChains>(
```

Comment:
Should we remove the `tokens` parameter or remove this method completely like we did for Algorand?